### ISSUE TITLE
Allow SubField position in YAML to be given in equatorial coordinates

### ISSUE THREAD

**Issue by denisGrBach** [IvS-KULeuven/PlatoSim3#422]
Today, the subField position is given in CCD-RF coordinates. It is therefore needed to calculate the pixel coordinates beforehand. This complex calculation needs to be consistent with the used input yaml-file and also with PlatoSim implementation itself which internally does the same calculation again. 

How about giving the wanted subField position it sky coordinates? Maybe optional. This would simplify it for the users who just want to see a particular star/sky region.

**Comment by SaraRegibo** [comment]
Hi Denis,
in `simulation.py` (in the `python` directory), you can find the method `setSubfieldAroundCoordinates(self, raStar, decStar, subfieldSizeX, subfieldSizeY, normal=True)`.  That does exactly what you need, I think!

**Comment by SaraRegibo** [comment]
Usage:

```
from simulation import Simulation

sim = Simulation(<some name for the simulation>)

# Configure as before, e.g. sim["CCD/CTI/Model"] = "Simple"

# (raStar, decStar): Sky coordinates of the centre of the sub-field [radians]
# (subfieldSizeX, subfieldSizeY): number of columns and rows in the sub-field [pixels]
# normal: True for normal cameras, False for fast cameras

sim.setSubfieldAroundCoordinates(raStar, decStar, subfieldSizeX, subfieldSizeY, normal=True)
```

**Comment by denisGrBach** [comment]
My point was that this call would not be needed if the SubField position can be given in sky coordinates directly. This would also avoid to calculate the same twice with phython/c++ code.

**Comment by SaraRegibo** [comment]
Oh, I thought you were not aware of the existence of this function! :)  At this moment, we use the sub-field **zeropoint** (i.e. lower left corner of the sub-field in the CCD reference frame) in pixel coordinates to specify the position of the sub-field on the CCD.  To me, when using sky coordinates, the logical thing to do is to specify the coordinates at the **centre** of the sub-field (rather than at the lower left corner).  Would you agree?

**Comment by denisGrBach** [comment]
indeed, it's basically the input of the phyton function (raStar, decStar, subfieldSizeX, subfieldSizeY). In fact, it would be useful to have both possibilities (pixel-/sky-coordinates).
